rpc/src/v1/types/uint.rs

```diff
@@ -48,6 +49,31 @@ macro_rules! impl_uint {
 			}
 		}
 
+		impl $name {
+			/// Human readable formatting
+			pub fn format_ether(&self) -> String {
```

Comment:
I'm not sure if this is the right place to put it `U256` is used in many other cases.
 It's used in two places within single file I think that `format_ether(val: U256)` in that file would suffice.